src/modules/simulator/accelsim/accelsim.cpp

```diff
@@ -318,65 +315,45 @@ ACCELSIM::~ACCELSIM()
 int
 ACCELSIM::init()
 {
-	int ret = -1;
-
-	struct mag_report mrp = {};
-	sensor_accel_s arp = {};
-
 	/* do SIM init first */
 	if (VirtDevObj::init() != 0) {
 		PX4_WARN("SIM init failed");
-		goto out;
+		return PX4_ERROR;
 	}
 
 	/* allocate basic report buffers */
 	_accel_reports = new ringbuffer::RingBuffer(2, sizeof(sensor_accel_s));
 
 	if (_accel_reports == nullptr) {
-		goto out;
+		return PX4_ERROR;
```

Comment:
```suggestion
		return -ENOMEM;
```

Reply:
Done!  Thanks!

Reply:
I added this same return style to gyrosim.cpp init() assuming it makes sense in that implementation as well.